lib/DDG/Spice/DebVersion.pm

```diff
@@ -0,0 +1,24 @@
+# Get the version of a package in various debian repositories
+package DDG::Spice::DebVersion;
+
+use Text::Trim;
+use DDG::Spice;
+
+triggers startend => "debian version", "debian versions", "deb ver", "debian ver";
+
+# spice from => '([^/]+)/?(?:([^/]+)/?(?:([^/]+)|)|)';
+spice to => 'http://sources.debian.net/api/src/$1/';
```

Comment:
Looks like this URL is redirecting to HTTPS--we don't support redirects in production so best to switch to https.